moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -534,8 +534,37 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
       title: "Interim MPD (Access) ID",
       field: "interim_project_id",
       hidden: hiddenColumns["interim_project_id"],
-      emptyValue: "-"
-    }
+      emptyValue: "-",
+    },
+    {
+      title: "Parent project",
+      field: "parent_project_id",
+      hidden: hiddenColumns["parent_project_id"],
```

Comment:
would you make both of these new columns hidden by default? would you also make these columns hidden by default as well?

```
construction_start_date
completion_end_date
project_tags
```